README.md

```diff
@@ -118,6 +118,7 @@ for tracking bugs and feature requests.
 * Ask a question on [Stack Overflow](https://stackoverflow.com/questions/tagged/aws-cdk)
   and tag it with `aws-cdk`
 * Come join the AWS CDK community on [Gitter](https://gitter.im/awslabs/aws-cdk)
+* You'll need an invite to join the [Slack](https://awsdevelopers.slack.com) community
```

Comment:
How about a slightly different wording, to make it more consistent with the Gitter one above?

```suggestion
* Talk in the CDK channel of the [AWS Developers Slack workspace](https://awsdevelopers.slack.com) (invite required)
```